code/game/objects/items/weapons/twohanded.dm

```diff
@@ -373,7 +373,8 @@
 		if(!isliving(target))
 			return
 		else
-			target.Weaken(4)
+			if(!istype(src, /obj/item/weapon/twohanded/chainsaw/rubber))

```

Comment:
Was this what you intended me to do, @tigercat2000?
Or did I misunderstand you again?